tests/test_mantatail.py

```diff
@@ -331,9 +331,12 @@ def test_channel_owner(user_alice, user_bob):
             break
 
 
-def test_operator_prefix(user_alice, user_bob, user_charlie):
+def test_founder_and_operator_prefix(user_alice, user_bob, user_charlie):
     user_alice.sendall(b"JOIN #foo\r\n")
-    time.sleep(0.1)
+    receive_line(user_alice)  # JOIN message from server
+
+    assert compare_if_word_match_in_any_order(receive_line(user_alice), b":mantatail 353 Alice = #foo :~Alice\r\n")
```

Comment:
There's only one nick, so the order should be exactly as in the tests and `compare_if_word_match_in_any_order` isn't needed:

```suggestion
    assert receive_line(user_alice) == b":mantatail 353 Alice = #foo :~Alice\r\n"
```